src/topology/order.lean

```diff
@@ -170,6 +170,29 @@ lemma generate_from_mono {α} {g₁ g₂ : set (set α)} (h : g₁ ⊆ g₂) :
 def tmp_complete_lattice {α : Type u} : complete_lattice (topological_space α) :=
 (gi_generate_from α).lift_complete_lattice
```

Comment:
While you're at it could we have a better name here? The `tmp` is I think referring to the fact this is only a "temporary" definition and we'll make another one later? Perhaps a further comment in the doc-string explaining the situation would solve the problem.

Reply:
I expanded the docstring, and also changed the position of the new material, so that `tmp_complete_lattice` is immediately followed by the construction of the "permanent" order (it was that way before, and I think that this provides a big hint to the reader about why it's `tmp`).

Reply:
Great.